packages/nimble-components/src/wafer-map/tests/ValueGenerator.ts

```diff
@@ -0,0 +1,13 @@
+export type IValueGenerator = (x: number, y: number) => number;
```

Comment:
update the file to use nimble's filename convention `value-generator.ts`

Reply:
applied